src/core/client/stream/tabs/Comments/Comment/ReplyCommentForm/CreateCommentReplyMutation.ts

```diff
@@ -179,6 +184,12 @@ function commit(
               username: viewer.username,
               createdAt: viewer.createdAt,
               ignoreable: false,
+              status: {
+                current: [GQLUSER_STATUS.ACTIVE],
+                ban: {
+                  active: false,
+                },
+              },
```

Comment:
Replace with `lookup` based on the viewer.

Reply:
This should be fixed up now.